cells/xschem/symbols/vnpn_10x10.sym

```diff
@@ -33,7 +33,7 @@ L 4 0 -10 20 -30 {}
 B 5 17.5 -32.5 22.5 -27.5 {name=C dir=inout pinnumber=3}
 B 5 -22.5 -2.5 -17.5 2.5 {name=B dir=in pinnumber=1}
 B 5 17.5 27.5 22.5 32.5 {name=E dir=inout pinnumber=2}
-B 5 17.5 -2.5 22.5 2.5 {name=B dir=in pinnumber=1}
+B 5 17.5 -2.5 22.5 2.5 {name=S dir=in pinnumber=1}
```

Comment:
@StefanSchippers is this change related?

Reply:
The latter one fixes a wrong pin in the vnpn transistor. It has the substrate pin incorrectly named as the base (B)  pin, making base and substrate shorted.
The fix renamed the source pin from 'B' to 'S'

Reply:
Make sense, updating the PR so that it makes it clearer that it fixes both before merging.